app/javascript/app/components/sectors-agriculture/countries-context/context-by-indicator/context-by-indicator-component.jsx

```diff
@@ -49,60 +50,105 @@ const ContextByIndicatorComponent = ({
   legend,
   tooltipTxt,
   countryData,
+  topTenCountries,
   updateIndicatorFilter,
   updateIndicatorYearFilter,
   handleCountryEnter
 }) => (
-  <React.Fragment>
-    <div className={styles.actionsContainer}>
-      <div className={styles.selectorWrapper}>
-        <Dropdown
-          label={'Indicator'}
-          value={selectedIndicator}
-          options={indicators}
-          onValueChange={updateIndicatorFilter}
-          hideResetButton
-        />
-      </div>
-      <div className={styles.timelineWrapper}>
-        <Dropdown
-          label={'Year'}
-          value={indicatorSelectedYear}
-          options={indicatorsYears}
-          onValueChange={updateIndicatorYearFilter}
-          hideResetButton
-        />
-      </div>
-      <div className={styles.buttonGroupWrapper}>
-        <ButtonGroup
-          className={styles.btnGroup}
-          buttonsConfig={buttonGroupConfig}
-        />
-      </div>
-    </div>
-    <Map
-      paths={paths}
-      tooltipId="cc-map-tooltip"
-      onCountryClick={() => {}}
-      onCountryEnter={handleCountryEnter}
-      onCountryFocus={() => {}}
-      dragEnable={false}
-    />
-    <MapLegend
-      mapColors={MAP_COLORS}
-      buckets={legend}
-      className={styles.legend}
-    />
-    {countryData && (
-      <ReactTooltip className={styles.tooltipContainer} id="cc-map-tooltip">
-        {getTooltip(countryData, tooltipTxt, selectedIndicator)}
-      </ReactTooltip>
+  <TabletLandscape>
+    {isTablet => (
+      <React.Fragment>
+        <div className={styles.actionsContainer}>
+          <Dropdown
+            label={'Indicator'}
+            value={selectedIndicator}
+            options={indicators}
+            onValueChange={updateIndicatorFilter}
+            hideResetButton
+          />
+          <Dropdown
+            label={'Year'}
+            value={indicatorSelectedYear}
+            options={indicatorsYears}
+            onValueChange={updateIndicatorYearFilter}
+            hideResetButton
+          />
+          <div className={styles.buttonGroupWrapper}>
+            <ButtonGroup
+              className={styles.btnGroup}
+              buttonsConfig={buttonGroupConfig}
+            />
+          </div>
+        </div>
+        <div className={styles.visualizationsContainer}>
+          <div>
```

Comment:
I think we don't need that extra div here

Reply:
It is used as container for Map and Legend, I've added a placeholder `layout-container` class to be explicit that they are a `column` layout element.